website/blog/2024-04-17-release-candidates.md

```diff
@@ -0,0 +1,34 @@
+---
+title: MLflow Release Candidates
+tags: [mlflow]
+slug: release-candidates
+authors: [mlflow-maintainers]
+thumbnail: img/blog/release-candidates.png
+---
+
+# Announcing MLflow Release Candidates
+
+We are excited to announce the implementation of a release candidate process for MLflow!
+The pace of feature development in MLflow is faster now than ever before, and the team has more exciting things planned in the near future. However, ensuring the stability of production systems is of paramount importance to us, and we want to make sure we’re moving fast without breaking things. With that goal in mind, we've decided to introduce a release candidate (RC) process. The RC process allows us to introduce new features and fixes in a controlled environment before they become part of the official release.
+
+## How It Works
+
+Starting from MLflow 2.13.0, new MLflow major and minor releases will be tagged as release candidates (e.g., `2.13.0rc0`) in PyPI two weeks before they are officially released.
+
+The release candidate process involves several key stages:
+
```

Comment:
- Feature Development Freeze: Prior to cutting the RC branch and announcing its availability, we will freeze the RC branch from feature commits. Once the branch is cut, only bug fix and stability PRs will be permitted to be merged, ensuring that unexpected, late-arriving, potentially regression-causing merges are not permitted to destabilize the forthcoming release.